packages/playground/ssr-react/__tests__/serve.js

```diff
@@ -45,9 +45,13 @@ exports.serve = async function serve(root, isProd) {
       const server = app.listen(port, () => {
         resolve({
           // for test teardown
-          close() {
-            server.close()
-            return vite && vite.close()
+          async close() {
+            await new Promise((resolve) => {
+              server.close(resolve)
+            })
+            if (vite) {
+              await vite.close()
+            }
```

Comment:
I think you can shorten this to 
```suggestion
            await new Promise((resolve) => server.close(resolve))
            await vite?.close()
```

Reply:
I didn't use an optional chain because of pre-Node-14 environments – is this file transpiled by Jest?

Reply:
ah okay, I'm so used to TypeScript ^^